components/apimgt/org.wso2.carbon.apimgt.common.analytics/pom.xml

```diff
@@ -98,4 +99,9 @@
             </plugin>
         </plugins>
     </build>
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <org.wso2.am.analytics.publisher.version>1.1.1-SNAPSHOT</org.wso2.am.analytics.publisher.version>
```

Comment:
Please use a released version. This is ok for testing but when propagating changes to master you will have to depend on a released version

Reply:
Ok Tishan..